nupic/algorithms/anomaly.py

```diff
@@ -157,6 +144,30 @@ def compute(self, activeColumns, predictedColumns,
     return score
 
 
+############################################
```

Comment:
Not problem to me on keep this separator (if really necessary) but please move it to the midle of the modules not above the next `def`. Something like this:

```
def funcion1():
  ...


##########


def function2():
   ..
```